app/models/pet.rb

```diff
@@ -3,10 +3,14 @@ class Pet < ApplicationRecord
   has_many :pet_applications, dependent: :destroy
   has_many :applications, through: :pet_applications
   before_save :default_values
+  validates_presence_of :name
+  validates_presence_of :approximate_age
+  validates_presence_of :sex
+  validates_presence_of :description
+
 
   def default_values
     self.adoption_status = "adoptable" if self.adoption_status.nil?
-    self.description = "Looking for a home" if self.description.nil?
```

Comment:
This is removed because of the migration I ran. It is now a default in the table.